cvat-core/src/annotations-objects.ts

```diff
@@ -2187,6 +2187,11 @@ export class MaskShape extends Shape {
 
     constructor(data: SerializedShape, clientID: number, color: string, injection: AnnotationInjection) {
         super(data, clientID, color, injection);
+        const [left, top, right, bottom] = this.points.slice(-4);
+        const { width, height } = this.frameMeta[this.frame];
```

Comment:
This line maybe within the condition below as width, height are not used anywhere else

Reply:
How this line can be put into the condition? The values width and height are used to determine if we need to execute the condition

Reply:
Oh, exactly